### ISSUE TITLE
NTR: Cysts on silver staining of nasopharyngeal secretions or pleural fluid (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9428]
New term request Cysts on silver staining of nasopharyngeal secretions or pleural fluid (nadea3_190611131940): Grocott stain

**Comment by pnrobinson** [comment]
@MickeySegal
Do we mean
Nasopharyngeal parasitic cysts
?
Possibly hydatid cysts?

**Comment by MickeySegal** [comment]
We use it for Pneumocystis pneumonia so we need the first form.

**Comment by pnrobinson** [comment]
Hm, I am seeing that Pneumocystis pneumonia may actuallybe a fungus rather that protozoa:
https://www.cdc.gov/dpdx/pneumocystis/index.html

Possibly it is better to have a term such as

Pneumocystis cysts in respiratory secretions
Def: Demonstration of cysts of Pneumocystis (a genus of related fungal species) in respiratory secretions such as nasopharyngeal secretions or pleural fluid.
PMID:25367973

thoughts?

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000256
- label: Pneumocystis cysts in respiratory secretions
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:25367973
- def: Demonstration of cysts of Pneumocystis (a genus of related fungal species) in respiratory secretions such as nasopharyngeal secretions or pleural fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9428
- synonyms:
